analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConstructorOverridableCall.cs

```diff
@@ -83,4 +87,24 @@ public override void DoSomething()
             Console.WriteLine($"{Name} is null at this point");
         }
     }
+
+    public class ClassB2 : Class1
```

Comment:
I suggest to add 2 additional cases.  One where the override is not sealed, however, the whole class is. And a second one where you don't even override the method and yet because the class is sealed you should still be safe.